qt/widgets/instrumentview/src/PeakOverlay.cpp

```diff
@@ -245,9 +245,13 @@ void PeakOverlay::createMarkers(const PeakMarker2D::Style &style) {
       return;
     }
     // Project the peak (detector) position onto u,v coords
-    double u, v, uscale, vscale;
-    m_surface->project(pos, u, v, uscale, vscale);
-
+    try {
+      double u, v, uscale, vscale;
```

Comment:
It looks like the builds are failing because these variables are defined inside the `try{}` but they are needed in the call to construct the peak marker.

Reply:
Ah yes sorry, I got a cpp warning when I had them above the try - so I'll just pop the marker stuff in the try as well!